ledger/internal/compactcert.go

```diff
@@ -140,40 +141,53 @@ func CompactCertParams(msg []byte, votersHdr bookkeeping.BlockHeader, hdr bookke
 	return
 }
 
+var (
+	errCompCertCrypto             = errors.New("compactcert crypto error")
+	errCompactCertParamCreation   = errors.New("compactcert param creation error")
+	errCompCertNotEnabled         = errors.New("compact certs not enabled")
+	errNotAtRightMultiple         = errors.New("cert is not in a valid round")
```

Comment:
should we include that the round is not a multiple for cert round in the message text?